producer-c/producer-cloudwatch-integ/KvsProducerSampleCloudwatch.cpp

```diff
@@ -162,13 +189,16 @@ INT32 main(INT32 argc, CHAR *argv[])
             frame.flags = frameIndex % DEFAULT_KEY_FRAME_INTERVAL == 0 ? FRAME_FLAG_KEY_FRAME : FRAME_FLAG_NONE;
 
             createCanaryFrameData(&frame);
-
             if (frame.flags == FRAME_FLAG_KEY_FRAME) {
                 if (lastKeyFrameTimestamp != 0) {
                     canaryStreamRecordFragmentEndSendTime(pCanaryStreamCallbacks, lastKeyFrameTimestamp, frame.presentationTs);
                     CHK_STATUS(computeStreamMetricsFromCanary(streamHandle, pCanaryStreamCallbacks));
                     CHK_STATUS(computeClientMetricsFromCanary(clientHandle, pCanaryStreamCallbacks));
                     currentMemoryAllocation(pCanaryStreamCallbacks);
+                    if((!fileLoggingEnabled) && (GETTIME() > currentTime + (60 * HUNDREDS_OF_NANOS_IN_A_SECOND))) {
```

Comment:
No need for extra parenthesis around !fileLoggingEnabled

Reply:
Added it for better readability.

Reply:
Sounds good then